src/Network/Packets.cs

```diff
@@ -41,1708 +42,4335 @@
 using ClassicUO.Game.GameObjects;
 using ClassicUO.Game.Managers;
 using ClassicUO.Game.UI.Gumps;
+using ClassicUO.IO;
 using ClassicUO.IO.Resources;
+using ClassicUO.Utility;
 
 namespace ClassicUO.Network
 {
-    internal sealed class PACKTalk : PacketWriter
+    static class NetClientExt
     {
-        public PACKTalk() : base(0x03)
-        {
-            WriteByte(0x20);
-            WriteByte(0x00);
-            WriteByte(0x34);
-            WriteByte(0x00);
-            WriteByte(0x03);
-            WriteByte(0xdb);
-            WriteByte(0x13);
-            WriteByte(0x14);
-            WriteByte(0x3f);
-            WriteByte(0x45);
-            WriteByte(0x2c);
-            WriteByte(0x58);
-            WriteByte(0x0f);
-            WriteByte(0x5d);
-            WriteByte(0x44);
-            WriteByte(0x2e);
-            WriteByte(0x50);
-            WriteByte(0x11);
-            WriteByte(0xdf);
-            WriteByte(0x75);
-            WriteByte(0x5c);
-            WriteByte(0xe0);
-            WriteByte(0x3e);
-            WriteByte(0x71);
-            WriteByte(0x4f);
-            WriteByte(0x31);
-            WriteByte(0x34);
-            WriteByte(0x05);
-            WriteByte(0x4e);
-            WriteByte(0x18);
-            WriteByte(0x1e);
-            WriteByte(0x72);
-            WriteByte(0x0f);
-            WriteByte(0x59);
-            WriteByte(0xad);
-            WriteByte(0xf5);
-            WriteByte(0x00);
-        }
-    }
-
-    internal sealed class PSeed : PacketWriter
-    {
-        public PSeed(byte[] version) : base(0xEF)
+        public static void Send_ACKTalk(this NetClient socket)
         {
-            const uint SEED = 0x1337BEEF;
-            WriteUInt(SEED);
+            const byte ID = 0x03;
 
-            for (int i = 0; i < 4; i++)
-            {
-                WriteUInt(version[i]);
-            }
-        }
+            int length = PacketsTable.GetPacketLength(ID);
 
-        public PSeed(uint v, byte major, byte minor, byte build, byte extra) : base(0xEF)
-        {
-            WriteUInt(v);
+            StackDataWriter writer = new StackDataWriter(length < 0 ? 64 : length);
+            {
+                writer.WriteUInt8(ID);
 
-            WriteUInt(major);
-            WriteUInt(minor);
-            WriteUInt(build);
-            WriteUInt(extra);
-        }
-    }
+                if (length < 0)
+                {
+                    writer.WriteUInt16BE(0x00);
+                }
 
-    internal sealed class PFirstLogin : PacketWriter
-    {
-        public PFirstLogin(string account, string password) : base(0x80)
-        {
-            WriteASCII(account, 30);
-            WriteASCII(password, 30);
-            WriteByte(0xFF);
-        }
-    }
+                writer.WriteUInt8(0x20);
```

Comment:
Fun fact, if you benchmark this against doing the same thing but with 8byte values, you will find the latter is much faster.